src/App.js

```diff
@@ -1,12 +1,51 @@
 import Title from './components/Title';
 import DatePicker from './components/DatePicker';
+import Spinner from './components/Spinner';
+import ErrorAlert from './components/ErrorAlert';
+import PictureInformation from './components/PictureInformation';
+import useFetch from './hooks/useFetch';
+import { apodUrl } from './utils/constants';
+import { useState } from 'react';
 
 const App = () => {
+  const [date, setDate] = useState(() => {
+    const dateObject = new Date();
+    const isoDate = dateObject.toISOString().split('T')[0];
+    return isoDate;
+  });
+  const [search, setSearch] = useState('');
```

Comment:
correct me if i'm wrong but i think this search term is just the date isn't it? Is it necessary to have both states?